pkg/cmd/util/output/backup_describer.go

```diff
@@ -261,6 +266,25 @@ func DescribeBackupStatus(d *Describer, backup *velerov1api.Backup, details bool
 	d.Printf("Persistent Volumes: <none included>\n")
 }
 
+func printBackupResourceList(d *Describer, backup *velerov1api.Backup, veleroClient clientset.Interface) {
```

Comment:
Now that I'm looking at this, I realize all the "print" functions are in the `backup_printer.go` file. Do you mind moving this as well as `printSnapshot` in there?

Reply:
I actually think these should stay in `backup_describer.go`, perhaps just rename to `describeBackupResourceList` and `describeSnapshot` for clarity.  The code that's in `backup_printer.go` is all for `velero backup get`, while `backup_describer.go` has the code for `velero backup describe` (which this is part of).

Reply:
Ah yes, I also agree that makes sense, looks like `_printer` always relates to a `get` command and `_describer` relates to `describe`. Slightly confusing naming, but I think renaming these functions to `describeBackupResourceList` as you suggested will help.